include/tvm/runtime/crt/graph_runtime.h

```diff
@@ -61,14 +61,20 @@ typedef struct TVMGraphRuntime TVMGraphRuntime;
  * \brief Allocate a new GraphRuntime with vmalloc and initialize it.
  *
  * \param sym_json JSON-encoded graph.
- * \param m TVM Module that exposes the functions to call.
+ * \param module_handle TVM Module that exposes the functions to call.
  * \param ctxs runtime execution context.
  */
-TVMGraphRuntime* TVMGraphRuntime_Create(const char* sym_json, const struct TVMModule* m,
+TVMGraphRuntime* TVMGraphRuntime_Create(const char* sym_json, TVMModuleHandle module_handle,
                                         const TVMContext* ctxs);
 
 int TVMGraphRuntime_GetInputIndex(TVMGraphRuntime* runtime, const char* name);
 
+/*!
+ * \brief get number of input tensors allocated.
+ * \return integer number of tensors available to use.
+ */
+int TVMGraphRuntime_GetNumInputs();
```

Comment:
Maybe its just me, but I cant see how this is relevant to linked params ?

Reply:
this one is needed for the CRT test. we didn't yet support the case of executing the graph runtime within the C runtime over RPC. test_linked_params uses this to verify that the CRT graph runtime can lookup the linked params. it's a bit tangential yes, but it is basically rounding out the graph runtime impl and it's not a very large addition; i'd prefer to keep it with this change to show the motivation.

Reply:
fine by me, then.